single_cell_RNAseq_snake/README.md

```diff
@@ -0,0 +1,72 @@
+# What
+
+This directory is meant to contain single cell pre-processing code using the snakemake workflow management system. 
+
+Snakemake is being evaluated as as alternative to nextflow.
+
+## Snakemake set-up
+
+### How was snakemake installed? 
+
+`mamba` is the recommended way to install snakemake. It is a conda distribution that some optimized features suitable for snakemake. 
+
+Miniforge is a repository that holds the minimal install for Conda.
+
+The conda distribution is installed in `/krummellab/data1/software/miniforge3` and the utilities like `mamba` live in: `/krummellab/data1/software/miniforge3/bin`.
```

Comment:
A lot of folks have their own conda version installed or use the `module load CBI conda` -- I dont know if this will present problems with using the distribution here, but we should check

Reply:
it shouldn't as each conda activation is its own isolated environment?

Reply:
each conda environment is, but the distribution is in a particular place. Folks who know more about conda might have answers to this? (Rebecca, Al)

Reply:
so it seems like this env var: `CONDA_EXE=/krummellab/data1/software/miniforge3/bin/conda` controls which distribution of conda you are pointing to.

I'm assuming other folks have this set to their own distributions.

Reply:
hmm a slight concern would be if others have directories with `conda` binaries in their `PATH` variable. Apparently when you run conda commands directly from the command line, the shell uses the first conda executable it finds in the PATH variable, following the order of directories listed in PATH.

Reply:
So I just tried this -- switching between my conda binary and this one -- and each time I switch I have to set the shell (conda init bash) which then writes to my .bashrc for that version and then restart the shell. Maybe there is a better way for a shared conda env? Can I go ahead and ask the group?